src/components/Prototypes/ParcelsAutoComplete.vue

```diff
@@ -475,7 +481,43 @@ export default {
       }, 100)
     },
 
-  },
+    validateUserInputPacage() {
+        let pacage = this.userInputPacage;
+        const params = {
+          numeroPacage: pacage,
+          years: [2019],
+        };
+
+        this.$store
+          .dispatch("operators/FETCH_WFS_LAYERS", params)
+          .then((data) => {
+            this.pacage = pacage;
+            this.campagne = 2019;
+            this.isLoading = false;
+            this.plots = this.formatFeatures(data[0][1]);
+          }, this);
```

Comment:
Y'a pas besoin de binder le `this` en utilisant des "arrow functions" (le scope est maintenu).